keepsorted/options_parser_test.go

```diff
@@ -224,6 +224,28 @@ func TestPopValue(t *testing.T) {
 			allowYAMLList: true,
 			want:          []*regexp.Regexp{regexp.MustCompile(".*"), regexp.MustCompile("abcd"), regexp.MustCompile("(?:efgh)ijkl")},
 		},
+		{
+			name: "Newlines",
```

Comment:
nit: I think these test cases should be named "IntOrBool" instead of "Newlines". This function is just testing the parsing of the IntOrBool type, and it doesn't actually know that it's used for the newline_separated option